cmd/restic/cmd_tag.go

```diff
@@ -115,23 +123,40 @@ func runTag(ctx context.Context, opts TagOptions, gopts GlobalOptions, args []st
 	defer unlock()
 
 	changeCnt := 0
+	changedSnapshots := make([]changedSnapshot, 0)
 	for sn := range FindFilteredSnapshots(ctx, repo, repo, &opts.SnapshotFilter, args) {
-		changed, err := changeTags(ctx, repo, sn, opts.SetTags.Flatten(), opts.AddTags.Flatten(), opts.RemoveTags.Flatten())
+		changedSnapshot, changed, err := changeTags(ctx, repo, sn, opts.SetTags.Flatten(), opts.AddTags.Flatten(), opts.RemoveTags.Flatten())
 		if err != nil {
 			Warnf("unable to modify the tags for snapshot ID %q, ignoring: %v\n", sn.ID(), err)
 			continue
 		}
 		if changed {
 			changeCnt++
+			changedSnapshots = append(changedSnapshots, *changedSnapshot)
 		}
 	}
 	if ctx.Err() != nil {
 		return ctx.Err()
 	}
+
 	if changeCnt == 0 {
 		Verbosef("no snapshots were modified\n")
 	} else {
 		Verbosef("modified tags on %v snapshots\n", changeCnt)
 	}
+
+	if gopts.JSON {
+		encoder := json.NewEncoder(gopts.stdout)
+		if encoder != nil {
+			err := encoder.Encode(changedSnapshots)
+			if err != nil {
+				return err
+			}
+		}
+	} else {
+		for _, changedSnapshot := range changedSnapshots {
+			Printf("%v -> %v\n", changedSnapshot.Original, changedSnapshot.New)
```

Comment:
Let's only print this information when the user specified `--verbose`. That is use `Verboseff` here.

The message also does not state what the value left and right of the arrow is.

Reply:
Done.